tests/test_config.py

```
@@ -0,0 +1,344 @@
+        """Test creating configuration template."""
+        with tempfile.NamedTemporaryFile(suffix='.yaml', delete=False) as f:
+            template_file = f.name
+        
+        try:
+            create_config_template(template_file)
+            
+            # Verify file was created
+            assert Path(template_file).exists()
+            
+            # Verify content is valid YAML
+            with open(template_file, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)